images/scripts/vms_entrypoint.sh

```diff
@@ -23,8 +25,25 @@ while [ "$MYINT" -lt "$INTFS" ]; do
   int_calc
 done
 
+
+ip link add link lan0 name macvtap0 type macvtap mode passthru
```

Comment:
I would left macvtap setup at create VM script, since logically they're required only for VMs. Also, creating all macvtaps could be redundant in case user wants to start up default flavor of mini-lab.

Reply:
I thought it would be a good idea to do it before doing the "ifup -a". Not sure if this was the cause why I could not remove the metal-core restart. If this was not the fix, we can of course move it back to the create vm script.

Reply:
It would totally make sense if that would be the reason) Let's leave it here then, until any additional evidence.